src/main/java/com/um/inventoryservice/DataLayer/OrderRepository.java

```diff
@@ -10,4 +10,5 @@ public interface OrderRepository extends ReactiveMongoRepository<Order, String>
 
     @Query("{'stockOrderDTOS.stockItemId': ?0}")
     Flux<Order> findOrdersByStockItemId(String stockItemId);
+    Flux<Order> findOrdersByClientId(String clientId);
```

Comment:
No query necessary for search by clientId?

Reply:
not necessary, stock item id was embedded in the list so we had to go get it with the query, client id is already an attribute of the order entity